helm/cluster-api/files/bootstrap/patches/versions/v1alpha2/kubeadmconfigs.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,566 @@
+- op: replace
```

Comment:
Kustomize patch with v1alpha2 OPEN API schema for `kubeadmconfigs.bootstrap.cluster.x-k8s.io`.